canvas_modules/common-canvas/src/common-properties/controls/expression/expression.scss

```diff
@@ -65,12 +66,28 @@
 			}
 		}
 		.validateLink {
-			min-height: 34px;
+			min-height: 32px;
 			padding: 4px;
 		}
 	}
 }
 
+.properties-expression-header {
+	position: relative;
+	height: 34px;
+	.properties-expression-title {
+		@include carbon--type-style("label-01");
+		position: absolute;
+		bottom: 5px;
+		left: 0;
+	}
+	.properties-expression-link-container {
+		position: absolute;
+		bottom: 0;
+		right: 0;
+	}
+}
+
 .properties-expression-button.bx--btn--sm.bx--btn--ghost {
 	padding: 4px;
```

Comment:
Why do we have padding here?  If this is needed we should use a token.

Reply:
I'm not sure when that was added, but it is needed, will replace it with a token.